### ISSUE TITLE
Uncaught exception when running runserver.py

### ISSUE THREAD

**Issue by HenrySia** [PokemonGoMap/PokemonGo-Map#1618]
## Current Behavior

> ./runserver.py 
> 2016-12-08 18:07:24,658 [      MainThread][     runserver][   ERROR] Uncaught exception
> Traceback (most recent call last):
>   File "./runserver.py", line 305, in <module>
>     main()
>   File "./runserver.py", line 168, in main
>     str(position[0]), str(position[1]))
> TypeError: 'NoneType' object has no attribute '__getitem__'

## Steps to Reproduce (for bugs)

1. git pull --recursive https://github.com/PokemonGoMap/PokemonGo-Map
2. Check you're on commit c8cb6468ed037c6b0fd8d265a745ce3f72863b95
3. npm install && npm run build
4. Modify config.ini, run ./runserver.py

## Your Environment

* Version used: c8cb6468ed037c6b0fd8d265a745ce3f72863b95
* Environment name and version (e.g. Python 2.7): Python 2.7
* Operating System and version (desktop or mobile): Ubuntu 16.04.1 LTS

**Comment by springjools** [comment]
Step 4 is incomplete: Modify config.ini, run ./runserver.py

Needs more info about what arguments are fed into the command line/config

**Comment by HenrySia** [comment]
Hi @springjools , here's my config.ini. I've been using the same one since last month.
Note: I removed my usernames, password, and gmaps-key.

```
# DO NOT USE NOTEPAD TO EDIT CONFIG FILES!! USE NOTEPAD ++ INSTEAD.
# Authentication settings
#auth-service:          # ptc (default) or google
username: [bla1, bla2, bla3]
password: secretpassword

# Database settings
#db-type: sqlite        # sqlite (default) or mysql
#db-host:               # required for mysql
#db-name:               # required for mysql
#db-user:               # required for mysql
#db-pass:               # required for mysql
#db-port:               # default 3306

# Search settings
location: Cal Anderson Park, Capitol Hill, WA
#no-gyms:               # disables gym scanning (default false)
#no-pokemon:            # disables pokemon scanning (default false)
#no-pokestops:          # disables pokestop scanning (default false)
#scan-delay:            # default 10
#step-limit:            # default 12
#gym-info:              # enables detailed gym info collection (default false)
#min-seconds-left:      # time that must be left on a spawn before considering it too late and skipping it (default 0)
#status-name:           # enables writing status updates to the database - if you use multiple processes, each needs a unique value

#Pokemon IV
#encounter:             # Set to true to start encounters to pull more info, like IVs or movesets. (default false)
#encounter-delay:       # delay in seconds before starting an encounter. Must not be zero. (default 1)
#encounter-whitelist:   # whitelist of pokemon ids to encounter. Syntax [id,id,id,id] (Do not use with blacklist)
#encounter-blacklist:   # blacklist of pokemon ids to NOT encounter. Syntax [id,id,id,id] (Do not use with whitelist)

# Misc
gmaps-key: VaLiD_GmApSkEy_HeRe             # your Google Maps API key
#proxy:                 # Proxy URL e.g. socks5://127.0.0.1:9050 or a list of proxies e.g. [socks5://127.0.0.1:9050,socks5://127.0.0.1:9050]
#proxy-timeout:         # Timeout before proceeding with next proxy while checking if the proxy works, (default 5)
#proxy-display:         # Used with -ps, full = display complete proxy address. Index = displays just the index for that proxy (default index)
#webhook:               # webhook URL (including http://)
#webhook-updates-only:  # only send updates to webhooks, (excludes gyms & non-lured pokéstops)

# Webserver settings
host: 0.0.0.0                 # address to listen on (default 127.0.0.1)
port: 7777                 # port to listen on (default 5000)
#locale:                # pokemon translation
#ssl-certificate:       # path to ssl certificate
#ssl-privatekey:        # path to ssl private key
#encrypt-lib:           # path to encrypt lib to be used instead of the shipped ones
#status-page-password:  # enables and protects the /status page to view status of all workers

#Uncomment a line when you want to change its default value (Remove # at the beginning)
#Please ensure to leave a space after the : (example setting: value)
#username, password, location and gmaps-key are required
```

**Comment by springjools** [comment]
I think it's this:
location: Cal Anderson Park, Capitol Hill, WA

Try to set it in quotes, and if that doesn'w work, use longitude and latitude, e.g.:
location: "Cal Anderson Park, Capitol Hill, WA"
location: "10.00, 51.00"

Maybe the first location doesn't work because the country is missing.

But yeah, that exception should be caught.

**Comment by sebastienvercammen** [comment]
Fixed with #1620.
